src/test/java/core/basesyntax/strategy/BalanceOperationTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.strategy.operations.BalanceOperationHandler;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationTest {
+    private Transaction testTransaction;
+    private OperationHandler balanceOperationTest;
+    private Fruit fruit;
+
+    @Before
+    public void setUp() {
+        testTransaction = new Transaction("b", new Fruit("apple"), 10);
+        balanceOperationTest = new BalanceOperationHandler();
+    }
+
+    @Test
+    public void balanceTestQuantity_Ok() {
+        fruit = new Fruit("apple");
+        balanceOperationTest.apply(testTransaction);
+        Integer expected = 10;
+        assertEquals(expected, Storage.storage.get(fruit));
+    }
+
+    @Test (expected = NullPointerException.class)
+    public void balanceGetEmptyValue_notOk() {
+        balanceOperationTest.apply(null);
+    }
+
+    @Test
+    public void noSuchOperationTest_notOk() {
+        fruit = new Fruit("papaia");
+        testTransaction = new Transaction("o", fruit, 200);
+        balanceOperationTest.apply(testTransaction);
+        assertEquals(1, Storage.storage.size());
+    }
```

Comment:
this test is not really relevant because handlers do not check type of operation
we can remove this test or consider this as ok behaviour